src/Runtime/Runtime/System.Windows/UIElement.EventManager.cs

```diff
@@ -189,6 +189,18 @@ private void ProcessOnMouseUp(object jsEventArg)
                     ProcessOnMouseRightButtonUp(jsEventArg);
                     break;
             }
+
+#if MIGRATION
+            if (IsMouseCaptured)
+            {
+                ReleaseMouseCapture();
+            }
+#else 
+            if (IsPointerCaptured)
+            {
+                ReleasePointerCapture();
+            }
+#endif
```

Comment:
This is going to work only if the captured element is the one for which the "mouse up" event is reported. But the captured element could be anything in the event route (or even outside if we release the mouse click outside of the element on which we pressed it initially)
```suggestion
            if (Pointer.INTERNAL_captured != null)
            {
#if MIGRATION
                Pointer.INTERNAL_captured.ReleaseMouseCapture();
#else
                Pointer.INTERNAL_captured.ReleasePointerCapture();
#endif
            }
```